src/cosmos-keystore.ts

```diff
@@ -38,15 +38,16 @@ export function storeWallet(
   name: string,
   password: string,
   network: string,
-  accountType: string,
+  HDPath: string,
+  curve: string,
```

Comment:
set a default here? this library only supports ed25519

Reply:
oh, ok